src/web/lib/hakuneko/frontend@classic-light/start.html

```diff
@@ -0,0 +1,108 @@
+<link rel="import" href="../../polymer/polymer-element.html">
+<link rel="import" href="../../polymer/lib/elements/dom-repeat.html">
+<link rel="import" href="../../polymer/lib/elements/dom-if.html">
+<link rel="import" href="theme.html">
+
+<dom-module id="hakuneko-start">
+    <template>
+        <style include="theme"></style>
+        <style>
+            #container {
+                width: calc(100% - 2em);
+                height: calc(100% - 2em);
+                padding: 1em;
+                background-color: var(--page-control-background-color);
+                background-image: var(--page-background-image);
+                background-size: cover;
+                background-attachment: scroll;
+                user-select: none;
+                font-size: 1.25em;
+                
+            }
+            .show {
+                display: block ;
+            }
+            .hide {
+                display: none;
+            }
+            h1 {
+                vertical-align:center;
+            }
+            h3 {
+                margin-bottom:0.1em;
+            }
+            .border {
+                border: 2px solid #cdcdcd;
+                border-color: rgba(0, 0, 0, .14);
+                border-radius: 1em;
+                background-color: var(--chapter-list-background-color);
+                padding: 1em;
+            }
+            .logo {
+                margin-right: 1em;
+                border-radius: 10%;
+            }
+        </style>
+        <div id="container">
+           <h3>Welcome to HakuNeko</h3>
+           <div class="whatis">
```

Comment:
Does this class already exist in a referenced resource?

Reply:
leftover from some design attemps. Removed